docs/getting-started-with-micro/index.md

```diff
@@ -0,0 +1,286 @@
+[INFO] EventLog - GOOD id:1608ca85-f5f9-4948-898a-728aa8f1131b app_id:test type:page_ping (iglu:com.snowplowanalytics.snowplow/page_ping/jsonschema/1-0-0)
+```
+
+This means your tracking is set up correctly and your events are valid (`GOOD`).
+
+If you wish to inspect the events further, you can use [the API](/docs/pipeline-components-and-applications/snowplow-micro/index.md). For example, try:
+
+```bash
+curl localhost:9090/micro/good
+```
+
+### Exporting events to TSV
+
+:::note
+
+This only applies to Micro 1.4.0 and above.
+
+:::
+
+Snowplow pipelines output data in the [_enriched TSV format_](/docs/understanding-your-pipeline/canonical-event/understanding-the-enriched-tsv-format/index.md). Typically, this is picked up by one of our [loaders](/docs/destinations/warehouses-and-lakes/index.md) or by tools such as [Snowbridge](/docs/destinations/forwarding-events/snowbridge/index.md).
+
+With Micro, you can see what your data would look like in this format — useful if you want to test any logic that is parsing this data. 
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv`
+}</CodeBlock>
+
+<details>
+<summary>Output vs logs</summary>
+
+The TSV data will be printed to the [standard output](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)). As you saw above, Micro also prints logs, which go into the [standard error stream](https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)).
+
+Depending on how you are running Micro, you might find the logs distracting. If so, you can turn off event logs with an extra option:
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  -Dorg.slf4j.simpleLogger.log.EventLog=off`
+}</CodeBlock>
+
+Or just discard the standard error stream entirely using the syntax appropriate for your shell:
+
+<CodeBlock language="bash">{
+`# for bash
+docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  2>/dev/null`
+}</CodeBlock>
+
+</details>
+
+## Advanced usage
+
+### Adding custom schemas
+
+There are two ways to add your own [schemas](/docs/understanding-tracking-design/understanding-schemas-and-validation/index.md) to Micro:
+* By pointing it to your own Iglu resolver
+* By loading your schema files directly into Micro
+
+<Tabs group="adding-schemas">
+  <TabItem value="resolver" label="Using a custom resolver" default>
+
+First, create your [Iglu resolver configuration](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md), including your custom resolver. Place it in `iglu.json`.
+
+<details>
+<summary>Example</summary>
+
+```json reference title="iglu.json"
+https://github.com/snowplow-incubator/snowplow-micro/blob/master/example/iglu.json
+```
```

Comment:
Something to upgrade in the micro repo, but we could be using 1-0-3 schema version.

Reply:
I am no longer linking here, so not my problem anymore™ :D But yes, we should update there.